packages/target-tauri/src-tauri/src/lib.rs

```diff
@@ -128,11 +130,15 @@ pub fn run() {
             temp_file::write_temp_file,
             temp_file::remove_temp_file,
             temp_file::copy_blob_file_to_internal_tmp_dir,
+            #[cfg(desktop)]
             webxdc::commands_main_window::on_webxdc_message_changed,
+            #[cfg(desktop)]
```

Comment:
It'd be nice to have it documented somewhere why these are desktop-only. Maybe here, or in a separate `.md` file.

Reply:
added a few comments